app/src/main/java/com/sunasterisk/moviedb_51/ui/home/adapter/GenreAdapter.kt

```diff
@@ -0,0 +1,44 @@
+package com.sunasterisk.moviedb_51.ui.home.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.recyclerview.widget.DiffUtil
+import com.sunasterisk.moviedb_51.BR
+import com.sunasterisk.moviedb_51.R
+import com.sunasterisk.moviedb_51.base.BaseRecyclerAdapter
+import com.sunasterisk.moviedb_51.base.BaseViewHolder
+import com.sunasterisk.moviedb_51.data.model.Genres
+import com.sunasterisk.moviedb_51.databinding.ItemGenreBinding
+
+class GenreAdapter : BaseRecyclerAdapter<Genres, GenreAdapter.ViewHolder>(MovieDiffUtilCallback()) {
+    var onItemClick: (Genres) -> Unit = { _ -> }
+
+    override fun getItemViewType(position: Int): Int = R.layout.item_genre
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val layoutInflater = LayoutInflater.from(parent.context)
+        val binding: ItemGenreBinding =
+            DataBindingUtil.inflate(layoutInflater, viewType, parent, false)
```

Comment:
viết như này được không 
```suggestion
        val binding = DataBindingUtil.inflate(layoutInflater, viewType, parent, false)
```

Reply:
Không được tới huy. không đủ thông tin